pyramid_openapi3/__init__.py

```diff
@@ -148,3 +151,23 @@ def spec_view(request: Request) -> FileResponse:
         }
 
     config.action(("pyramid_openapi3_spec",), register, order=PHASE0_CONFIG)
+
+
+def check_all_routes(event: ApplicationCreated):
+    """Asserts all routes in the spec are defined on the app.
```

Comment:
I'd like to avoid adding more `.flake8` ignores, so I'd have `Assert` instead of `Asserts` here. Is there a reason why not to?

Reply:
Done.